goagen/gen_client/cli_generator.go

```diff
@@ -162,6 +162,9 @@ func (g *Generator) generateCommands(commandsFile string, clientPkg string, func
 		}
 		return res.IterateActions(func(action *design.ActionDefinition) error {
 			name := codegen.Goify(action.Name, false)
+			// To avoid underscore in the resource name, we apply the kebabCase
+			// algorithm over the action.Parent.Name property.
+			action.Parent.Name = codegen.KebabCase(action.Parent.Name)
```

Comment:
I think a better approach here would be to leave the name of the resource and action unchanged (they could be used for other things now and in the future) and instead call the `KebabCase` function from the template where the command and action names are defined (see `registerCmdsT`).